src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1416,10 +1416,13 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
                         await writer.WriteAsync("efghij");
                         s.Shutdown(SocketShutdown.Send);
 
-                        // The task should complete and the request content should be disposed
+                        // The task should complete and the request content should be disposed, but only on desktop.
+                        // On core, we explicitly made a change to not dispose of the content, as doing so doesn't
+                        // behave well with various handlers, in particular those that allow response processing
+                        // even before the full request may have been sent.
                         using (HttpResponseMessage response = await post)
                         {
-                            Assert.True(contentDisposed, "Expected request content to be disposed");
+                            Assert.Equal(PlatformDetection.IsFullFramework, contentDisposed);
```

Comment:
This test is always going to fail.  Since System.Net.Http OOBs on Desktop, these tests never actually test inbox System.Net.Http.dll.  They test the OOB on Desktop.